### PR TITLE
Remove unused code from cfglib.py

### PR DESCRIPTION
**Pull request by SVilgelm**

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
Consider resolving pylint issues: \
``` \
************* Module fabfile
fabfile.py:265: [W0612(unused-variable), load_yaml_config] Unused variable 'name'

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
can be merged

File: cfglib.py [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+
 from oslo.config import cfg
-import addons
```
Comment (miarmak) [inline cfglib.py]: I guess in this case we can delete `addons` directory from the CloudFerry root directory as well. Anyway plugins system has not been implemented properly...

**Pull request merged**